e2e/keywords/node.resource

```diff
@@ -0,0 +1,121 @@
+*** Settings ***
+Documentation     
+...               Keywords defined here can be used where this Keywords.resource in loaded.
+Library    ../libs/steps.py
+Resource    ../keywords/utility.resource
+
+*** Variables ***
+${POD_EVICTION_TIME_OUT}=    300
+
+*** Keywords ***
+The ${size} GB ${volume_type} volume with ${request_replica_count} replicas
```

Comment:
```suggestion
A ${size} GB ${volume_type} volume with ${request_replica_count} replicas
```
Should also make this extensible. Ref to [my previous comment](https://github.com/longhorn/longhorn-tests/pull/1300#discussion_r1211185591).